tools/SourceKit/include/SourceKit/Support/Logging.h

```diff
@@ -109,9 +109,9 @@ class Logger : public llvm::RefCountedBase<Logger> {
 #define LOG_FUNC_SECTION_WARN LOG_FUNC_SECTION(Warning)
 
 #define LOG(NAME, LEVEL, msg) LOG_SECTION(NAME, LEVEL) \
-  do { *Log << msg; } while (0)
+  do { *Log << msg << '\n'; } while (0)
```

Comment:
I wonder if this should move into `Logger::~Logger` so that it applies to all LOG_SECTION.  We could also check dynamically there whether log message already ended with a newline. @akyrtzi any opinion?

Reply:
> I wonder if this should move into Logger::~Logger

Sounds like a good idea!

> We could also check dynamically there whether log message already ended with a newline

I don't think it's necessary, I'd prefer to keep it simple.

Reply:
Oh, I'm strongly in favour of that idea.  Thanks!

BTW, this should fix the following logging that I se:
```
sourcekit: [2:canUseASTWithSnapshots: 8.7614] will try existing ASTsourcekit: [2:handleRequest-after: 8.7619] {
```